pkg/query-service/app/http_handler.go

```diff
@@ -2192,6 +2193,130 @@ func (aH *APIHandler) WriteJSON(w http.ResponseWriter, r *http.Request, response
 	w.Write(resp)
 }
 
+// Preferences
+func (ah *APIHandler) RegisterPreferenceRoutes(router *mux.Router, am *AuthMiddleware) {
```

Comment:
These routes seem restfully wrong.

An organisation/user has preferences. The path to get preferences should be
- user/preferences for all preferences of a user
- user/preferences/{preferenceId} for an individual preference

Similarly for org.

Reply:
updated